bindings/node/test/types/index.test-d.ts

```diff
@@ -1,63 +0,0 @@
-import { expectAssignable, expectError, expectType, expectNotType, expectNotAssignable } from 'tsd';
-import { RangeOptions, AWSEncryptionKeyOptions, AzureEncryptionKeyOptions, ClientEncryption, GCPEncryptionKeyOptions, ClientEncryptionEncryptOptions, KMSProviders } from '../..';
-
-type RequiredCreateEncryptedCollectionSettings = Parameters<
-  ClientEncryption['createEncryptedCollection']
->[2];
-
-expectError<RequiredCreateEncryptedCollectionSettings>({});
-expectError<RequiredCreateEncryptedCollectionSettings>({
-  provider: 'blah!',
-  createCollectionOptions: { encryptedFields: {} }
-});
-expectError<RequiredCreateEncryptedCollectionSettings>({
-  provider: 'aws',
-  createCollectionOptions: {}
-});
-expectError<RequiredCreateEncryptedCollectionSettings>({
-  provider: 'aws',
-  createCollectionOptions: { encryptedFields: null }
-});
-
-expectAssignable<RequiredCreateEncryptedCollectionSettings>({
-  provider: 'aws',
-  createCollectionOptions: { encryptedFields: {} }
-});
-expectAssignable<RequiredCreateEncryptedCollectionSettings>({
-  provider: 'aws',
-  createCollectionOptions: { encryptedFields: {} },
-  masterKey: { } as AWSEncryptionKeyOptions | AzureEncryptionKeyOptions | GCPEncryptionKeyOptions
-});
-
-{
-  // NODE-5041 - incorrect spelling of rangeOpts in typescript definitions
-  const options = {} as ClientEncryptionEncryptOptions;
-  expectType<RangeOptions | undefined>(options.rangeOptions)
-}
-
-{
-  // KMSProviders
-  // aws
-  expectAssignable<KMSProviders['aws']>({ accessKeyId: '', secretAccessKey: '' });
-  expectAssignable<KMSProviders['aws']>({ accessKeyId: '', secretAccessKey: '', sessionToken: undefined });
-  expectAssignable<KMSProviders['aws']>({ accessKeyId: '', secretAccessKey: '', sessionToken: '' });
-  // automatic
-  expectAssignable<KMSProviders['aws']>({});
-
-  // azure
-  expectAssignable<KMSProviders['azure']>({ tenantId: 'a', clientId: 'a', clientSecret: 'a' });
-  expectAssignable<KMSProviders['azure']>({ tenantId: 'a', clientId: 'a', clientSecret: 'a' });
```

Comment:
I don't think tests for TS that is effectively constant hold value, so I'm not going to add them unless that's something the team decides it wants.

I also don't think the release test is necessary, since it's an internal package after the next major release.